src/test/java/hudson/plugins/jacoco/JacocoPublisherTest.java

```diff
@@ -276,9 +277,9 @@ public void testResolveFilePathsAbstractBuildException() throws Exception {
 	@Test
 	public void testLocateReports() throws Exception {
 		// Create a temporary workspace in the system
-		File w = File.createTempFile("workspace", ".test");
-		assertTrue(w.delete());
-		assertTrue(w.mkdir());
+		File w = Files.createTempDirectory("workspace" + ".test").toFile();
+		assertTrue(true);
+		assertTrue(true);
```

Comment:
@JLLeitschuh , `assertTrue(true)` looks weird

Reply:
I agree, it's a bug I thought had been fixed:
https://github.com/openrewrite/rewrite-java-security/commit/27b5dc1322fdfb169d82ffa8b367b39bbd301179

Reply:
You and I are on the same page